spec/config-spec.coffee

```diff
@@ -1287,6 +1290,54 @@ describe "Config", ->
           atom.config.set 'foo.bar', ['2', '3', '4']
           expect(atom.config.get('foo.bar')).toEqual  [2, 3, 4]
 
+      describe 'when the value has a "color" type', ->
+        beforeEach ->
+          schema =
+            type: 'color'
+            default: 'white'
+          atom.config.setSchema('foo.bar.aColor', schema)
+
+        it 'coerces various types to a color object', ->
+          atom.config.set('foo.bar.aColor', 'red')
+          expect(atom.config.get('foo.bar.aColor')).toEqual {red: 255, green: 0, blue: 0, alpha: 1}
```

Comment:
> I imagine the 99% case is that someone will want conversion to hex or rgba string

Yeah, I went back and forth about how much to put in this object initially.

I've update it to use a `Color` class with `toHexString` and `toRGBAString` methods on it.